js-classes-inheritance/script.js

```diff
@@ -0,0 +1,94 @@
+class Builder {
+    constructor (arg) {
+        this.arg = arg;
+    }
+
+    plus(...n) {
+        for (let i = 0; i < arguments.length; i++) {
+            this.arg += arguments[i];
+        }
+        return this;
+    }
+
+    minus(...n) {
+        if (typeof this.arg == "string") {
+            this.arg = this.arg.slice(0, this.arg.length - n);
+            return this;
+        }
+        for (let i = 0; i < arguments.length; i++) {
```

Comment:
Again try to use declarative approach, which means I want to sum this numbers (using for example reduce) instead of imperative.

https://ui.dev/imperative-vs-declarative-programming/